features/cli/oc_expose.feature

```diff
@@ -50,12 +50,13 @@ Feature: oc_expose.feature
       | resource | secret/ssl-key   |
     Then the step should succeed
     Given evaluation of `File.read("tls.crt")` is stored in the :crt clipboard
-    And evaluation of `secret('ssl-key').created` is stored in the :birth clipboard
+    And evaluation of `v = secret('ssl-key').created; (v.is_a? Time) ? v : Time.parse(v)` is stored in the :birth clipboard
     And evaluation of `Time.now` is stored in the :t2 clipboard
     And evaluation of `(cb.birth + (cb.t2 - cb.t1) + 3600 + 60).utc.strftime "%Y-%m-%dT%H:%M:%SZ"` is stored in the :newexpiry clipboard
     When I run the :annotate client command with:
       | resource     | secret/ssl-key                                          |
       | keyval       | service.alpha.openshift.io/expiry=<%= cb.newexpiry %>   |
+      | keyval       | service.beta.openshift.io/expiry=<%= cb.newexpiry %>    |
```

Comment:
Now `oc get secret ssl-key -o yaml` shows both:
```
  annotations:
    service.alpha.openshift.io/expiry: "..."
    service.beta.openshift.io/expiry: "..."
```
Above fix is compatible no matter alpha or beta.

Reply:
Without above fix, there is failure:
```
expression "File.read("tls.crt") != cb.crt" returned non-positive status: false
RuntimeError
/home/jenkins/workspace/Runner-v3/features/step_definitions/common.rb:136:in `/^(?:the )?expression should be true> (.+)$/'
/home/jenkins/workspace/Runner-v3/features/step_definitions/meta_steps.rb:48:in `block (2 levels) in '
/home/jenkins/workspace/Runner-v3/lib/base_helper.rb:153:in `wait_for'
/home/jenkins/workspace/Runner-v3/features/step_definitions/meta_steps.rb:43:in `/^I wait(?: up to ([0-9]+|<%=.+?%>) seconds)? for the steps to pass:$/'
features/cli/oc_expose.feature:69:in `Given I wait up to 1800 seconds for the steps to pass:'
```